connectors/sources/jira.py

```diff
@@ -730,6 +750,8 @@ async def _verify_projects(self):
         """
         if self.jira_client.projects == ["*"]:
             return
+
+        self._logger.info("Verifying the configured projects")
```

Comment:
Could we include the projects here?

Reply:
Yes, we can include it here to get more idea which projects are being verified here.